pkgs/by-name/mu/multipass/package.nix

```
@@ -0,0 +1,64 @@
+{
+  callPackage,
+  fetchFromGitHub,
+  lib,
+  nixosTests,
+  symlinkJoin,
+}:
+
+let
+  name = "multipass";
+  version = "1.15.0-rc4";
+
+  multipass_src = fetchFromGitHub {
+    owner = "canonical";
+    repo = "multipass";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-xwEM7Zsjkyi91oNyiX0i1mIJfBOoJ/4X47JBEqP8xYY=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
+    postFetch = ''
+      # Workaround for https://github.com/NixOS/nixpkgs/issues/8567
+      cd $out
+      rm -rf .git
+    '';
+  };
+
+  commonMeta = with lib; {
+    homepage = "https://multipass.run";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ jnsgruk ];
+    platforms = [ "x86_64-linux" ];
+  };
+
+  multipassd = callPackage ./multipassd.nix {
+    inherit multipass_src version;
+    meta = commonMeta // {
+      description = "Backend server & client for managing on-demand Ubuntu VMs";
+    };
+  };
+
+  multipass-gui = callPackage ./gui.nix {
+    inherit multipassd multipass_src version;
+    meta = commonMeta // {
+      description = "Flutter frontend application for managing Ubuntu VMs";
+    };
+  };
+in
+symlinkJoin {
+  inherit version;
+  pname = name;
+
+  paths = [
+    multipassd
+    multipass-gui
+  ];
+
+  passthru.tests = {
+    multipass = nixosTests.multipass;
+  };
```

```suggestion
  passthru.tests = lib.optionalAttrs stdenv.hostPlatform.isLinux {
    inherit (nixosTests) multipass;
  };
```
Making this conditional stops OfBorg from trying to evaluate this on incompatible platforms and always failing

---

Fixed, thanks :)